### DISCUSSION TITLE
Beta 8.2.1.6a1 for Firefox - should fix truncated video

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#236] [Announcements]
- If you get truncated videos, I hope this version will fix the issue.
- Some MPD/Dash content will now be shown as HLS.

Install it here: https://downloadhelper.net/data/vdh-mozilla-beta-latest.xpi

**Comment by Mattmjs** [comment]
Yes looks like you got it, but on one random one from vimeo there is an error : https://vimeo.com/337424927

**Reply by paulrouget** [reply]
Thank you! Looking at it now.

**Reply by paulrouget** [reply]
Hmm, Vimeo via HLS is not always successful. Gonna offer both (MPD/Dash & HLS).

**Reply by paulrouget** [reply]
@Mattmjs see https://github.com/aclap-dev/video-downloadhelper/discussions/237

**Reply by Mattmjs** [reply]
Ok, I see both HLS and Dash, that might be an extreme case as the video is 5 years old. It has an error with HLS but Dash works.
But it finished 2 seconds short of the video. Not too bad. There are other HLS manifests which do work but they don't appear
directly in the network.

**Reply by paulrouget** [reply]
>  There are other HLS manifests which do work but they don't appear
directly in the network.

What do you mean?

**Reply by Mattmjs** [reply]
Well on vimeo.com you can find them like this :

![Screenshot 2024-03-01 203545](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/03cbdc76-af1d-4390-84ef-3e03a95528d2)

Open that link in a new tab, scroll down to find some m3u8 manifests which work.

![Screenshot 2024-03-01 203842](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/974347b2-4a6b-4d2c-8138-8328577b2b09)

If they're embedded on other sites, the m3u8 manifests are found in the Response of the html document

![Screenshot 2024-03-01 204959](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/28557793-8696-4b2f-8de2-3872445ac26a)

Whether this practical or not I don't know.

**Reply by paulrouget** [reply]
Well. I totally didn't know about that config file. I've always use a very different method that gives me MPD & M3U8. Maybe that config file will make the M3U8 detection more reliable!

I'll give it a try. Thank you.

**Reply by ebguy06** [reply]
seems to be working but my stream downloads are 1/4 the speed from before all these issues.
ex. a 3:47 video, downloaded 248mb in that amount of time; now getting 389mb in 27:00min~!...
[when you divide total mb per sec, ratio is about 1/4 the speed.] 

is this an issue with the co app? because the streaming video rate for 1080 was 10mbs before, now it says 8.8mbs, as well as the "time" on the download continues to increase. in just 20min, my download says 1% , 7+ hrs, 8.8mbs.
@40min, download modified in dwhelper folder, it says its still downloading stream, but 389mb in 40min is not correct, it should have downloaded in several GB's of data.  Timer now says "1% 130hrs+ 8.8mbs" on the 1080 stream.

Still testing todays beta release, not looking promising. 
When I go to the dwhelper folder, normally while downloading I can hit F5 to see how many mb have been downloading; downloadhelper still saying its downloading, but for 30min, when I hit refresh, the file size hasnt changed, this was noticeable since 2/14 update.


2/13/2024 was the last dated recording I have that worked flawless. 8.85mbs data rate. 
latest stable ver, getting 736kbs, starts off at 8.85mbs than drops for 2-3min into kbs, downloaded file modifies and stops.


how do i go back to older version before 2/13?

Ive tried Version 8.2.0.23; Version 8.1.4.0; Version 8.0.0.6, including old co apps. VdhCoApp1.6.3 VdhCoApp 2.0.19
_over 15 hours of testing, I give up... insane, been over two weeks and I cant get it back to prior to 2/14._

`
version: 8.2.2.0a1
channel: beta
build date: 2024-03-01
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\oneof",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
`

**Reply by GrampaWildWilly** [reply]
You shouldn't keep your problematic videos secret.  As long as a login is not required, free or paid, you should always post URLs so other people can try to replicate your results.  If a login is required, then you need to provide screenshots of what is happening.
